### ISSUE TITLE
Display pokemon statistics retrieved from the json

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#28]
Once the result is displayed, we can display stats of each pokemon on hovering over it's sprite.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by deadbirdi** [comment]
I'll take this one

**Comment by jmeline** [comment]
I'll give it a shot if you don't mind. Looks fun.
